main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func check(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	m := make(map[string]string)
+	a := len(names)
+	for i := a - 1; i >= 0; i-- {
+		m[names[i]] = numbers[i]
+	}
+	return m
+}
+func main() {
+	var names, numbers []string
+	f, err := os.Open("phone_numbers.txt")
+	check(err)
+	scanner := bufio.NewScanner(f)
+	var str []string
```

Comment:
An extra string slice is not required here. Please change the implementation to not use it.